public/styles/styles.css

```diff
@@ -0,0 +1,20 @@
+
+@import "./nav.css";
+@import "./list.css";
+@import "./detail.css";
+@import "./update.css";
+@import "./create.css";
+
+body {
```

Comment:
cq--
I would move body and header to a different file and for consistency keep here only @import here